cassandra_db/csvGenerator.js

```diff
@@ -0,0 +1,51 @@
+const fs = require('fs');
+const faker = require('faker');
+const Uuid = require('cassandra-driver').types.Uuid;
```

Comment:
This looks a little odd, though I understand why. Consider using ES6 object destructuring (example below), though I recognize it doesn't make it look all that much better...  :
```suggestion
const { types: { Uuid }} = require('cassandra-driver');
```